config.yml

```diff
@@ -11,7 +11,16 @@ receivers:
         endpoint: 0.0.0.0:4317
 
 processors:
-  batch:
```

Comment:
does this work? It looks like it might break, with the colon on the end of batch it'd expect something indented on the next line?